### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by apezio**
the line ending in '$amount_ato$' was cutoff.

The RPC URL is now passed into verify.php via POST, instead of hardcoding it.

$gatewaymodule = "Monero"; in verify.php - "M" is now lowercase (was throwing an error).

### CODE REVIEW

File: modules/gateways/monero/verify.php [hunk 1]
```diff
@@ -6,39 +6,44 @@
...
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
+
+//$gatewaymodule = "monero";
+//$GATEWAY = getGatewayVariables($gatewaymodule);
+
 $invoice_id = $_POST['invoice_id'];
 $payment_id = $_POST['payment_id'];
 $amount = $_POST['amount'];
+$link = $_POST['link'];
```
Comment (serhack) [inline modules/gateways/monero/verify.php]: We can't change link of wallet rpc! That will be a security issue

Comment (apezio) [reply modules/gateways/monero/verify.php]: okay.  How to get the variable from the module config into verify.php?  I tried using getGatewayVariables() but couldnt make it work. :(

**Comment by serhack** [comment]
Can you test it? Thanks

**Comment by serhack** [comment]
Strange, have you insert those in your admin settings?

**Comment by apezio** [comment]
Yes, it works in createinvoice.php just not verify.php. probably due to how verify.php is loaded.  Has it worked for you?

**Comment by serhack** [comment]
We can't send the url, because someone can post the wrong url and confirm the Payment!!!!

**Pull request merged**